.travis.yml

```diff
@@ -15,8 +15,9 @@ deploy:
   cache-control: "max-age=31536000"
   local_dir: dist
   skip_cleanup: true
+  acl: public_read
```

Comment:
This is required to make the uploaded js files available in Digital Ocean spaces